src/chmhtmlwindow.cpp

```diff
@@ -28,12 +28,29 @@
 #include <hhcparser.h>
 #include <memory>
 #include <wx/clipbrd.h>
+#include <wx/dnd.h>
 #include <wx/filename.h>
 #include <wx/log.h>
 #include <wx/uri.h>
 #include <wx/wfstream.h>
 #include <wx/wx.h>
 
+class CHMDropTarget : public wxFileDropTarget {
+    CHMFrame* m_frame;
+
+public:
+    CHMDropTarget(CHMFrame* frame) : m_frame(frame) {}
+
+    bool OnDropFiles(wxCoord x, wxCoord y, const wxArrayString& filenames) override
```

Comment:
This causes an unused parameters build warning, please comment out the parameter names to fix it (e.g. `wxCoord /* x */, wxCoord /* y */`).